src/components/molecules/BasicButton/styles.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components"
+import { colors, fsize, fweight } from "../../../styles/vars"
+
+export const StyleBasicButton = styled.button`
+    background-color: ${colors.red.normal};
+    border-radius: 50px;
+    color: #fff;
+    font-size: ${fsize.big};
+    font-weight: ${fweight.w7};
+    height: 40px;
```

Comment:
do we need the height?

Reply:
When I remove the height property, the button shrinks. Using min-height also works, so is that better?